### PR TITLE
Update to ruby 2.5

### PR DESCRIPTION
**Pull request by grodowski**
Started this, because I got ruby 2.4 parser errors on one of the Rainforest PRs. I initially called this PR "Update to ruby 2.6.3", but turns out 2.5 might be a good intermediate step.

What updates are required for 2.6?
- `rubocop 0.6x`
- `rf-stylez`
- `pronto` and all supporting gems to `0.10.0`
- also `pronto-commentator` (RF gem) needs to be updated to match the new pronto version: https://github.com/rainforestapp/pronto-commentator/pull/4

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 WORKDIR /app
+
+RUN gem update --system
+RUN gem install bundler:1.17.2
+
 COPY . .
 RUN bundle install
-ENTRYPOINT ["circlemator"]
+
+ENTRYPOINT ["bundle exec circlemator"]
```
Comment (grodowski) [inline Dockerfile]: Something changed in `2.6` and running the binstub without `bundle exec` produces this error:

```
Traceback (most recent call last):
	2: from /usr/local/bundle/bin/circlemator:23:in `<main>'
	1: from /usr/local/lib/ruby/site_ruby/2.6.0/rubygems.rb:305:in `activate_bin_path'
/usr/local/lib/ruby/site_ruby/2.6.0/rubygems.rb:286:in `find_spec_for_exe': can't find gem circlemator (>= 0.a) with executable circlemator (Gem::GemNotFoundException)
```

I tried updating rubygems and recreating the bundle but it didn't help. Any ideas why this happens?

Comment (grodowski) [reply Dockerfile]: Also applies to `2.5`

Comment (kruszczynski) [reply Dockerfile]: It might be related to `bundler`'s update to 2

Comment (jbarber) [reply Dockerfile]: /shrug

Comment (jbarber) [reply Dockerfile]: It might be because you're getting bundle >2 and it's strange?

Comment (grodowski) [reply Dockerfile]: Turns out it is irrelevant to the bundler version. To dig a bit more I pulled the original docker image for `rainforestapp/circlemator` and updated rubygems from `2.7.4` to `3.0.4`, which produced the same error. `rake install:local` generates correct binstubs though so I was able to remove the `bundle exec` from the entrypoint in the final setup.

**Pull request merged**